### ISSUE TITLE
Possible to pass-through some keys?

### ISSUE THREAD

**Issue by ngoonee** [the-cavalry/light-locker#65]
Any possibility to allow some keys through (either configurable or a switch for 'allow' or 'don't allow')?

For example, I use sxhkd (xbindkeys etc., there's lots of options) and have bindings for media keys. Obviously those don't make it through light-locker. Would be cool if they could and I didn't have to remember to turn off locking when using the laptop as a media player.

**Comment by ochosi** [comment]
You could create a separate user account for your media usage and disable locking there. I think passing through of keys is exactly what a locker should prevent.

Also, there are media players that can still be controlled with the computer being locked, e.g. mpd through a smartphone in the same wifi or some other computer.

**Comment by ngoonee** [comment]
Yeah wifi control is possible, but inconvenient. So is a separate user account (if I knew beforehand that I'd be away that long, I'd just use xset to disable the screen off).

The specific (corner?) use case I've encountered a few times this past week is that I'd be on my laptop with music playing and would turn away to do some paperwork, play with my son, etc. And then I'd need to pause the music or skip a song, and I'd find I needed to enter my password to do that. Completely understand if light-locker doesn't support that, its not within it's job scope, but it seems to me to be a valid use case which separate user accounts or wifi control does not solve.

On the Arch Linux forums someone suggested that I could use acpid to monitor specific keys which DO go through lockscreens, though this means I'd have to relearn the keys I use for media control. This last paragraph is just left here in case someone stumbles on this issue through google.

**Comment by de-live-gdev** [comment]
Would be great if this can be implemented as an Opt-In. Managing Screen Brightness (Laptops,...) and Media Keys (For audio playback -> +- Volume, Mute, NextPreviousPause) should really be available at lock screen.
Really disturbing to always need to enter password to do these basic things. I know a lockscreen is for preventing unothrizes access, but thats not what this is about. I can control screen and volume at desktops by switching on/off the audio system, but thats not possible on laptops.  
Note that at least those 2 things at top are available/implemented at least at gnome-screensaver and Windows (at least since W7).

**Comment by MrHaroldA** [comment]
I was expecting this to be a feature. If I lock my laptop go fetch a coffee, I often forget to stop my media player.

I now have to unlock my system to pause my music ...
